code/game/machinery/sprinkler.dm

```diff
@@ -0,0 +1,214 @@
+/obj/machinery/power/plumbing
```

Comment:
Sprinklers don’t need to be /power. The reason the pump was machinery/power/ was to make it work of off wires, so it could be used in lavaland (unpowered)

I see the conflict here, and ideally being wireable should just be a machinery property. Just keep it as is, I’ll move some stuff to the plumbing component and have exceptions handled there.